cpp/velox/shuffle/VeloxShuffleWriter.cc

```diff
@@ -1420,26 +1398,31 @@ arrow::Status VeloxShuffleWriter::splitFixedWidthValueBuffer(const facebook::vel
       *actual = 0;
       return arrow::Status::OK();
     }
-    EvictGuard{splitState_};
```

Comment:
> EvictGuard{splitState_};

This didn't seem to guard anything. The correct form IIUC is 

```
EvictGuard guard{splitState_};
```
? Was this the reason the previous patch didn't work? Or by some other reasons?